<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="img_src">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>161</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>src</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="img_preset_1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>151</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>preset1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="img_preset_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>151</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>preset2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="img_preset_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>120</y>
      <width>151</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>preset3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="merge">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>450</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>합        성</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select_src">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>select_src</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
